File: test13_out/test13/HelloWorld/HelloWorld.js

/*
 * ======== MODULE test13.HelloWorld ========
 */

var $$c = function() {
    const HelloWorld = {};
    HelloWorld.HelloWorld = HelloWorld
    HelloWorld.$name = 'test13.HelloWorld';
    HelloWorld.pollen$used = 0;
    HelloWorld.Event = $units['test13.Event'];
    HelloWorld.ClockSource_SCLK = 2;
    HelloWorld.ClockSource_ACLK = 7;
    HelloWorld.start = new $$Ref('test13_HelloWorld_start__I');
    HelloWorld.pollen__run = new $$Ref('test13_HelloWorld_pollen__run__E');
    HelloWorld.handle = new $$Ref('test13_HelloWorld_handle__E');
    HelloWorld.targetInit = new $$Ref('test13_HelloWorld_targetInit__I');
    HelloWorld.test = function() {
        var $$text = '';
        var x;
        var z;
        x = (HelloWorld.ClockSource_SCLK);
        z = ($units['test13.YEnum'].YEnum_Y1);
        return $$text;
    }
    HelloWorld.$$hostInit = function() {
        var $$text = '';
        return $$text;
    }
    HelloWorld.pollen__uses__ = function() {
        $$bind($units['test13.YEnum'], 'pollen$used', true);
        $$bind($units['test13.Event'], 'pollen$used', true);
    }
    HelloWorld.$$privateInit = function() {
    }
    return HelloWorld;
}
var $$u = $$c();
$units['test13.HelloWorld'] = $$u;
$units.push($$u);

File: test13_out/test13/HelloWorld/HelloWorld.c

/*
 * ======== string literals ========
 */

static const char test13_HelloWorld__s__fileName[] = "HelloWorld.p";
static const char test13_HelloWorld__s__0[] = "HelloWorld";

/*
 * ======== function definitions ========
 */

void test13_HelloWorld_pollen__run__E() {
    test13_HelloWorld_start__I((test13_HelloWorld_e1__V));
    test13_HelloWorld_start__I((test13_HelloWorld_e2__V));
}

bool test13_HelloWorld_start__I( struct test13_Event*  e ) {
    return(  true  );
}

void test13_HelloWorld_handle__E( struct test13_Event*  y ) {
    uint8 x;
    uint8 z;
    x = (test13_HelloWorld_ClockSource_SCLK);
    z = (test13_YEnum_YEnum_Y1);
}

void test13_HelloWorld_targetInit__I() {
}

